analysis/test/getFiles

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Moving into data folder #
+if [ "$HOSTNAME" == "ui10.sdfarm.kr" ] || [ "$HOSTNAME" == "ui20.sdfarm.kr" ]
+then
+   cd /cms/scratch/`whoami`/nanoAOD/src/nano/analysis/data/
+   # Downloading Lumimask txt #
+   echo "Downloading LumiMask text file into : `pwd`"
+   wget https://cms-service-dqm.web.cern.ch/cms-service-dqm/CAF/certification/Collisions16/13TeV/Final/Cert_271036-284044_13TeV_PromptReco_Collisions16_JSON.txt
+
+   # Copying Rochester txt #
+   echo "Copying Rochester correction txt file into : `pwd`"
+   cp -r /cms/scratch/UOS/Roccor/rcdata.2016.v3 ./
+
+   # Moving into Btag SF folder #
+   cd /cms/scratch/`whoami`/nanoAOD/src/nano/analysis/data/btagSF/
```

Comment:
this is already stored in nanoAOD-tools... 
git clone https://github.com/cms-nanoAOD/nanoAOD-tools.git PhysicsTools/NanoAODTools